src/Button.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const Button = props => {
+  const {
+    isActive,
+    onClick,
+    children,
+  } = props;
+  
+  return (
+    <button
+      className={`btn ${isActive ? "is-active" : ""}`}
```

Comment:
inside curly brackets(`{}`) in JSX we write JS, strings should be in single quotes